server/src/dev/resources/logback-spring.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@akosyakov do we need to have this file here?

Reply:
Why do we have it back? The task is only address @amvanbaren commits for mirror-feature branch. @filiptronicek You should not rebase this branch on master and so on. It was already cleaned up. It is another step to rebase gitpod-io:master on eclipse:master afterwards.